app/main.py

```diff
@@ -16,4 +16,23 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+line = [
+    lucky_number,
+    pi,
+    one_is_a_prime_number,
+    name,
+    my_favourite_films,
+    profile_info,
+    marks,
+    collection_of_coins
+]
+
+sorted_variables = dict(mutable=[], immutable=[])
+
+mutable_types = [list, dict, set]
+
+for i in line:
+    if type(i) in mutable_types:
```

Comment:
```suggestion
    if isinstance(i, tuple(mutable_types)):
```
You can use isinstance

Reply:
Why for? I thought about this func, but imho hardreading. 
Is this important func?